templates/errors/error.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'layouts/_base.html' %}
+
+{% block main %}
+<h1>{{ heading }}</h1>
+{%- if retry_message -%}
+  <p data-qa="retry">{{ retry_message }}</p>
+{%- endif -%}
+<p>{{ contact_us_message.format(contact_us_url=contact_us_url)  }}</p>
+{% endblock %}
```

Comment:
Waiting on confirmation from Laura to see if this template is enough to handle both IR and confirmation email as currently I stripped out 1 line.

Reply:
Got confirmation from Laura and Andy that for IR, it can just be `If this problem keeps happening, please contact us for help.` so the new template is fine.